### ISSUE TITLE
DataGrid pager styling changed in 0.9.4

### ISSUE THREAD

**Issue by naefp** [Megabit/Blazorise#2679]
**Describe the bug**
Not sure if this is a bug or planned to work like that.

With updating Blazorise to `0.9.4` the styling of the DataGrid pager changed. In previews version the page-link elements inherited the color, now I had to add something like that to get back the old behavior:
```css
/* old behavior for pager style after DataGrid update */
.page-link, .page-link:hover {
    color: inherit;
}
```

just close the issue, if that was done on purpose.

**Comment by David-Moreira** [comment]
@stsrki rings a bell? Anything new in this department? Want me to take a look?

**Comment by stsrki** [comment]
I think I actually fixed and this is good.

If I remember it correctly, theme settings were not applied for pagination links. So I added new CSS rules in the theme generator.

**Comment by naefp** [comment]
Just for my understanding, this fix should come with 0.9.4.2, right?
I am asking, because it is not mentioned in #2660 and with 0.9.4.1 I still had to override this.

**Comment by stsrki** [comment]
Yes, it will be in 0.9.4.2. 

0.9.4.1 is already out so no way to re-release it.
